src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                                rdfs:subClassOf soreag:GeologicBoundary ;
+                                rdfs:label "mohorovicic discontinuity"@en .
+
+
+###  http://sweetontology.net/realmGeol/Monocline
+soreag:Monocline rdf:type owl:Class ;
+                 rdfs:subClassOf soreag:StructuralFeature ;
+                 dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                 dcterms:source <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+                 skos:definition "A one-limbed flexure on either side of which the strata are horizontal or dip uniformly at low angles. (Allaby, 2013)"@en ;
+                 skos:prefLabel "monocline"@en .
+
+
+###  http://sweetontology.net/realmGeol/MyloniteZone
+soreag:MyloniteZone rdf:type owl:Class ;
+                    rdfs:subClassOf soreag:StructuralFeature ;
+                    dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                    dcterms:source <http://www.glossaryofgeology.org> ,
+                                   <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+                    skos:definition "Usually found in narrow, planar zones of localised ductile deformation, although kilometre-scale zones exist. Mylonites form in shear zones by mechanical grinding, crushing and recrystallization of rock to produce a metamorphic rock that has foliation and is much finer grained than its precursor. (Modified from: Neuendorf et al., 2011; Allaby, 2013)"@en ;
+                    skos:prefLabel "mylonite zone"@en .
+
+
+###  http://sweetontology.net/realmGeol/Nappe
+soreag:Nappe rdf:type owl:Class ;
+             rdfs:subClassOf soreag:TechtonostratigraphicEntity ;
+             dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+             dcterms:source <http://www.glossaryofgeology.org> ,
+                            <https://doi.org/10.1093/acref/9780199653065.001.0001> ;
+             skos:definition "A sheetlike, allochthonous rock unit, which has moved on a predominantly horizontal surface as a result of thrust faulting, recumbent folding, or both.  Fold limbs and axes are approximately horizontal (After Neuendorf et al., 2011; Allaby, 2013)"@en ;
+             skos:prefLabel "nappe"@en .
+
+
+###  http://sweetontology.net/realmGeol/NappeComplex
+soreag:NappeComplex rdf:type owl:Class ;
+                    rdfs:subClassOf soreag:TechtonostratigraphicEntity ;
+                    dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                    dcterms:source <https://doi.org/10.1080/11035897.2016.1178666> ;
+                    skos:definition "A nappe complex consists of two or more distinct nappes that are stacked upon, and related to, one another in some significant respect, e.g., shared general environment of origin, similar metamorphism and/or deformation. (Kumpulainen, 2017)"@en ;
+                    skos:prefLabel "nappe complex"@en .
```

Comment:
not a definition.
is a stratigraphic feature a kind of geologic unit?  See discussion of 'stratigraphicEvent' in the comments. 

NOTE:  The current SWEET ontolgoy does not seem to have a 'GeologicUnit' concept, which is central to the GeoSciML conceptual model.  soreag:LithodemicFeature, soreag:StratigraphicFeature, soreag:GeophysicalFeature all appear to me to be kinds of GeologicUnit.